x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/import_rules_route.ts

```diff
@@ -187,7 +196,11 @@ export const importRulesRoute = (router: IRouter, config: ConfigType, ml: SetupP
 
                   throwHttpError(await mlAuthz.validateRuleType(type));
 
-                  const rule = await readRules({ alertsClient, ruleId, id: undefined });
+                  let rule;
```

Comment:
You could get rid of this `let` by doing something like: `const rule = rules?.find((aRule) => aRule.params.ruleId === ruleId);`